windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -96,6 +119,13 @@ namespace winrt::ReactNativeWebView::implementation {
                 WriteWebViewNavigationEventArg(webView, eventDataWriter);
                 eventDataWriter.WriteObjectEnd();
             });
+
+        if (m_messagingEnabled) {
+            winrt::hstring windowAlert = L"window.alert = function (msg) {window.chrome.webview.postMessage(`{\"type\":\"__alert\",\"message\":\"${msg}\"}`)};";
+            winrt::hstring postMessage = L"window.ReactNativeWebView = {postMessage: function (data) {window.chrome.webview.postMessage(String(data))}};";
```

Comment:
1) when passing strings that have embedded quotes, use raw string literals to make them easier to write / read: https://docs.microsoft.com/en-us/cpp/cpp/string-and-character-literals-cpp?view=msvc-170

2) don't create unnecessary HSTRINGs, put everything in one :)

3) in this case it seems like it doesn't matter but in general you might want to await the ExecuteScriptAsync to make sure the control has processed adding the alert / RNWV properties to the window object.

Reply:
This section was just copied from the original WebView code. Seems like a good change to make though.